integration/manualtests/conn_test.go

```diff
@@ -0,0 +1,69 @@
+package manualtests
+
```

Comment:
I guess one of these files was meant to be deleted

Reply:
Yep. thanks!